src/services/connections.js

```diff
@@ -235,9 +235,8 @@ module.exports = class ConnectionHelper {
 				}
 			})
 
-			//To be removed later
-			connectionsWithDetails.forEach((detail) => {
-				detail.user_details.image = 'https://picsum.photos/200'
+			connectionsWithDetails.forEach(async (detail) => {
+				detail.user_details.image = await utils.getDownloadableUrl(detail.user_details.image)
```

Comment:
@nevil-mathew download the image from user service

Reply:
We're using the newly added image in user extension to get the image URL.